m2cgen/assemblers/xgboost.py

```diff
@@ -0,0 +1,106 @@
+import json
+import numpy as np
+from m2cgen import ast
+from m2cgen.assemblers import utils
+from m2cgen.assemblers.base import ModelAssembler
+
+
+class XGBoostModelAssembler(ModelAssembler):
+
+    def __init__(self, model):
+        super().__init__(model)
+        self._base_score = self.model.base_score
+        self._n_estimators = self.model.n_estimators
+
+        feature_names = enumerate(self.model.get_booster().feature_names)
+        self._feature_name_to_idx = {name: idx for idx, name in feature_names}
```

Comment:
Nit: Usually `enumerate` is used in the loop, but I will leave it it's up to you.

Reply:
Not sure I understand. Isn't this a loop?

Reply:
```python
feature_names = self.model.get_booster().feature_names
self._feature_name_to_idx = {
    name: idx for idx, name in enumerate(feature_names)}
```

Otherwise you need to remeber that `feature_names` is already result of `enumerate`.

Reply:
Fixed